app/validators/article_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ArticleValidator < ActiveModel::Validator
+  def validate(record)
+    unless Article.find_by(short_path: record.short_path, id: record.id)
```

Comment:
So i'm not understanding the logic around how to make the validation on redirect know if it exists on the article. Any thoughts on that?